src/docs/release/breaking-changes/parent-data-widget-generic-type.md

```diff
@@ -0,0 +1,119 @@
+---
+title: The generic type of ParentDataWidget changed to ParentData
+description: The ParentDataWidget is now bound to the ParentData type.
+---
+
+# Generic type of `ParentDataWidget` changed to `ParentData`
```

Comment:
Can you pull the latest into your repo? I just landed a PR that updates the template. We don't want a level 1 header, since the website generates that automatically. Instead, I'd like to see a level 2 "Summary" header.